docs/examples/widgets/month_calendar.py

```diff
@@ -0,0 +1,12 @@
+from textual.app import App, ComposeResult
+from textual.widgets import MonthCalendar
+
+
+class MonthCalendarApp(App):
```

Comment:
Adding a `CSS` variable containing the following CSS to this app yields better results. I reckon we should add the equivalent CSS to the widget itself to override the default DataTable behaviour (unless you can think of a reason not to).

```scss
    MonthCalendar { height: auto; width: auto; }
    MonthCalendar > DataTable { height: auto; width: auto; }
```

We don't do this in the `DataTable` itself because it's intended to scroll, but this widget has a very constrained height compared to the `DataTable` which can be arbitrarily large.

Reply:
Sorry I should have mentioned, I haven't started looking at the `DEFAULT_CSS` yet as I wanted to focus on the core functionality first. Do you think the table/columns widths should just be static, or allow the developer to specify the column width or grow dynamically according to the widget width?

Reply:
I've now added some basic `DEFAULT_CSS` in cfcac14. I've flagged `min-width` as TODO as [this might depend on  locale](https://github.com/Textualize/textual/pull/3667#discussion_r1393350712)?